src/main/resources/application.properties

```diff
@@ -1,3 +1,6 @@
 spring.datasource.driverClassName=org.h2.Driver
 spring.h2.console.enabled=true
 spring.h2.console.path=/h2-console
+spring.datasource.url=jdbc:h2:mem:testdb
```

Comment:
These lines were added due to an issue with accessing the local H2 console, where username and password were incorrect.
![unknown](https://user-images.githubusercontent.com/68877945/158034045-a8e7f3a0-aff4-416b-a7b0-573aab9414d8.png)